apps/marketplace/actions/appActions.js

```diff
@@ -51,12 +52,20 @@ export function setAuthState(newState) {
   return { type: SET_AUTH, newState }
 }
 
+export function setAuthFrameworkError(frameworkError) {
+  return { type: SET_AUTH_FRAMEWORK_ERROR, frameworkError }
+}
+
 export const fetchAuth = () => dispatch => {
   dispatch(sendingRequest(true))
+  dispatch(setAuthFrameworkError(false))
   dmapi({ url: '/ping' }).then(response => {
     if (response.error) {
       dispatch(setErrorMessage(GENERAL_ERROR))
     } else {
+      if (response.data.framework === 'orams') {
```

Comment:
safer to do !== 'digital-marketplace' - we could fix it if we introduce a 3rd app